src/api/spec/factories/attribs.rb

```diff
@@ -48,6 +48,12 @@
       values { [build(:attrib_value, value: Faker::Lorem.sentence)] }
     end
 
+    factory :auto_cleanup_attrib do
+      attrib_type { AttribType.find_by_namespace_and_name!('OBS', 'AutoCleanup') }
+
+      values { [build(:attrib_value, value: 14)] }
```

Comment:
the bug is there - OBS:Autocleanup contains a timestamp. And '14' was parsed on 2019-11-28 as a date in the past (11-14) while today it's parsed as a date in the future (12-14)

As such the test case is flickering - just very slow, it will work again on 15th :)